### PR TITLE
remove outdated copyright statements

### PR DESCRIPTION
**Pull request by matthiasdiener**

### CODE REVIEW

File: logpyle/__init__.py [hunk 1]
```diff
@@ -33,8 +33,6 @@
 .. autofunction:: add_simulation_quantities
 """
 
-__copyright__ = "Copyright (C) 2009-2013 Andreas Kloeckner"
```
Comment (inducer) [inline logpyle/__init__.py:36]: There's no reason to remove this. Copyright statements serve to track file authorship and don't get modified, only added to.

Comment (matthiasdiener) [reply logpyle/__init__.py:36]: Ok, so should this get the same line as in the LICENSE file?

Comment (inducer) [reply logpyle/__init__.py:36]: It records distinct information from the LICENSE file. Technically, everyone contributing to each file should make sure their copyright is included at the top of the file. (Or, for CEESD, that the board of trustees copyright is included).

Comment (matthiasdiener) [reply logpyle/__init__.py]: Ok, removed this change for now.

**Review by inducer — APPROVED** [review]

**Pull request merged**